src/Nest/Mapping/Types/Properties-Scalar.cs

```diff
@@ -178,6 +181,7 @@ public partial class PropertiesDescriptor<T> : IsADictionaryDescriptorBase<Prope
 
 		public PropertiesDescriptor<T> Scalar(Expression<Func<T, string>> field, Func<TextPropertyDescriptor<T>, ITextProperty> selector) =>
 			SetProperty(selector?.InvokeOrDefault(new TextPropertyDescriptor<T>().Name(field)));
```

Comment:
This should do the same thing as string inference on automapping i.e. add a keyword multi_field